src/core/db/connection.py

```diff
@@ -11,105 +8,104 @@
 Any new backend must implement the DataHubConnection interface
 '''
 
+
 class DataHubConnection:
-  def __init__(self, user, password, repo_base=None):
-    self.backend = PGBackend(user, password, repo_base=repo_base)
-  
-  def reset_connection(self, repo_base):
-    self.backend.reset_connection(repo_base=repo_base)
 
-  def close_connection(self):    
-    self.backend.close_connection()
+    def __init__(self, user, password, repo_base=None):
+        self.backend = PGBackend(user, password, repo_base=repo_base)
 
-  def create_repo(self, repo):
-    return self.backend.create_repo(repo=repo)
+    def reset_connection(self, repo_base):
+        self.backend.reset_connection(repo_base=repo_base)
 
-  def list_repos(self):
-    return self.backend.list_repos()
+    def close_connection(self):
+        self.backend.close_connection()
 
-  def delete_repo(self, repo, force=False):
-    return self.backend.delete_repo(repo=repo, force=force)
+    def create_repo(self, repo):
+        return self.backend.create_repo(repo=repo)
 
-  def add_collaborator(self, repo, username, privileges, auto_in_future=True):
-    return self.backend.add_collaborator(
-        repo=repo,
-        username=username,
-        privileges=privileges,
-        auto_in_future=auto_in_future)
+    def list_repos(self):
+        return self.backend.list_repos()
 
-  def delete_collaborator(self, repo, username):
-    return self.backend.delete_collaborator(repo=repo, username=username)
+    def delete_repo(self, repo, force=False):
+        return self.backend.delete_repo(repo=repo, force=force)
 
-  def list_tables(self, repo):
-    return self.backend.list_tables(repo=repo)
+    def add_collaborator(self, repo, username,
+                         privileges, auto_in_future=True):
```

Comment:
yes - the variable was passed to add_collaborator in manager.py and pg.py, but pg.py didn't make any use of it. I figured that when it because useful, we'd add it back in. Otherwise, it's just confusing to pass around variables that don't do anything.